File: C++/C++Basic/07.For-Loop-Lab/09.leftAndRightSum.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;
    cin >> n;

    int leftSum = 0;
    int rightSum = 0;

    for (int i = 0; i < n; i++)
    {
        int num;
        cin >> num;

        leftSum += num;
    }

    for (int i = 0; i < n; i++)
    {
        int number;
        cin >> number;
        
        rightSum += number;
    }

    if (leftSum == rightSum){
        cout << "Yes, sum = " << leftSum << endl;
    }
    else {
        int diff = abs(leftSum - rightSum);
        
        cout << "No, diff = " << diff << endl;
    }

    return 0;
    
    

}

File: Python/Basic/07.For-Loop-Lab/09.left_and_right_sum.py
n = int(input())

left_sum = 0
right_sum = 0

for idx in range(n * 2):
    number = int(input())

    if idx < n:
        left_sum += number
    else:
        right_sum += number

if left_sum == right_sum:
    print(f'Yes, sum = {left_sum}')
else:
    print(f'No, diff = {abs(left_sum - right_sum)}')